ui/src/app/base/components/SSHKeyForm/SSHKeyFormFields/SSHKeyFormFields.tsx

```diff
@@ -76,7 +76,11 @@ export const SSHKeyFormFields = ({
           </p>
         </Col>
       </Row>
-      <Link href="https://maas.io/docs/user-accounts#heading--ssh-keys">
+      <Link
+        href="https://maas.io/docs/user-accounts#heading--ssh-keys"
+        rel="noreferrer"
```

Comment:
we should probably include `noopener` for all of these links too even though they link to maas.io, juuuuust to be safe

Reply:
Done.